### DISCUSSION TITLE
Publish prisma-client-rust and prisma-client-rust-cli to crates

### DISCUSSION THREAD

**Discussion by listepo** [Brendonovich/prisma-client-rust#433] [Ideas]
Make possible  `cargo install prisma-client-rust-cli`
```
  provider      = "prisma-client-rust-cli"
```

**Comment by Brendonovich** [comment]
https://github.com/Brendonovich/prisma-client-rust/issues/76
https://github.com/Brendonovich/prisma-client-rust/issues/307
https://github.com/Brendonovich/prisma-client-rust/issues/392
